client/oci_env/utils.py

```diff
@@ -64,6 +64,9 @@ def get_config(env_file):
         "DJANGO_SUPERUSER_USERNAME": "admin",
         "DJANGO_SUPERUSER_PASSWORD": "password",
 
+        # Pytest fixtures
+        "REMOTE_FIXTURES_ORIGIN": "https://fixtures.pulpproject.org/",
+
```

Comment:
So this value is the default anyway. Seems like we only needed to set this in case we run the local fixtures container. Would you mind turning this pr into a profile that spins up and configures that local container in the pod?

Reply:
Will try.